docs/user/installation_and_configuration.rst

```diff
@@ -459,3 +475,12 @@ is printed out when you start GeoServer::
     ----------------------------------
     - GEOSERVER_DATA_DIR: /opt/devel/install/geoserver-data-dir
     ----------------------------------
+
+Note about Apache Commons Collections
+-------------------------------------
+
+Version 3.2.1 and earlier of the Apache Commons Collections library have a CVSS 10.0 vulnerability.  Read more `here
+https://commons.apache.org/proper/commons-collections/security-reports.html`__.
+
+Note that Accumulo 1.6.5 is the first version of Accumulo which addresses this security concern.
```

Comment:
If they're binary compatible, then I would suggest we upgrade to 1.6.5 in our POM.  That way, we're sending a consistent message (at least on this single, small point).  It's not a democracy, but if it were, that's how I'd vote.